lib/mix/lib/mix.ex

```diff
@@ -148,7 +148,9 @@ defmodule Mix do
   Where `&clean_extra/1` would be a function in your `mix.exs`
   with extra clean up logic.
 
-  Note aliases do not show up on `mix help`.
+  Note: 
+    * aliases do not show up on `mix help`
+    * aliases are not merged in from the `mix.ex` files of dependent project definitions.
```

Comment:
Can we write this as a sentence instead of a list?

A dependent project would be a parent project, I think you mean a dependencies project definitions.